server/alpine.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM frolvlad/alpine-glibc:latest
+
+LABEL varion="1.0" \
+    maintainer="h1dden-da3m0n" \
+    description="A simple TeaSpeak server running on alpine-glibc (amd64)"
+
+ARG TEASPEAK_VERSION="1.4.12"
+
+RUN	apk update --no-cache && apk upgrade --no-cache \
+    && apk add --no-cache \
+         bash ca-certificates wget curl python libnice ffmpeg tzdata \
+    \
+    && mkdir ts && cd /ts \
+    && wget -nv -O TeaSpeak.tar.gz https://repo.teaspeak.de/server/linux/amd64/TeaSpeak-$TEASPEAK_VERSION.tar.gz \
+    && tar -xzf TeaSpeak.tar.gz \
+    && rm TeaSpeak.tar.gz \
+    && curl -sLS https://yt-dl.org/downloads/latest/youtube-dl -o /usr/local/bin/youtube-dl \
+    && chmod a+rx /usr/local/bin/youtube-dl
+
+WORKDIR /ts
+
+EXPOSE 9987/udp 10101/tcp 30303
```

Comment:
When exposing ports here, is there really a need to put them into the command line again?
(As well web-client required port 9987/tcp)

Reply:
Short answer: Yes.
Long answer: 
To define the ports in the Dockerfile is a best practice, it shows what ports are going to be used by the app inside of the container. This WON'T expose the here listed ports at runtime though unless you use the `-P` option in the cli run command. The `-P` option would expose all ports that have been defined in the Image/Dockerfile.
So a separate listing of the ports in the `run` command is necessary to even export the ports from the container through to the host.
Otherwise, without that, those ports would only be reachable in the docker bridge network that the container is in by default.

Reply:
I'll add a docker-compose demo to the readme too, so users can choose to use either the docker `run` cli or use docker-compose

Reply:
Well what kind of crap is that? That's not really making any sense to  add such an configuration tag when its, more or less, "cosmetic" (in general docker perspective).
But its still good to have them here.

I never worked with Docker compose, but yeahr might be helpful ;)